src/NuGet.Core/NuGet.Versioning/PublicAPI.Shipped.txt

```diff
@@ -62,7 +62,7 @@ NuGet.Versioning.SemanticVersionConverter.SemanticVersionConverter() -> void
 NuGet.Versioning.VersionComparer
 NuGet.Versioning.VersionComparer.Compare(NuGet.Versioning.SemanticVersion? x, NuGet.Versioning.SemanticVersion? y) -> int
 NuGet.Versioning.VersionComparer.Equals(NuGet.Versioning.SemanticVersion? x, NuGet.Versioning.SemanticVersion? y) -> bool
-NuGet.Versioning.VersionComparer.GetHashCode(NuGet.Versioning.SemanticVersion! version) -> int
+NuGet.Versioning.VersionComparer.GetHashCode(NuGet.Versioning.SemanticVersion? version) -> int
```

Comment:
I still think we should prefer API design over implementation and not make this nullable. 0 is a valid hash code for a non-null instance, and I think we should encourage API customers to not request nonsensical values. When I checkout this branch locally and build there are no build errors.